docs/components/cache.rst

```diff
@@ -1,4 +1,122 @@
-Cache
+CACHE
 #####
 
+Symfony makes heavy use of a filesystem cache. Frequently clearing this cache will be required when developing for Mautic. By default, the cache is located in ``app/cache/ENV`` where ``ENV`` is the environment currently accessed (``dev`` or ``prod``). To rebuild the cache, the ``ENV`` can just be deleted or run the Symfony command ``php app/console cache:clear --env=ENV``. If a specific environment is not passed to the command via ``--env=ENV``, the ``dev`` environment will be used by default.
 
+In the ``dev`` environment, translations, views, and assets are not cached. However, changes to these files will require the cache to be cleared for them to take effect in the ``prod`` environment. Changes to Mautic config files, Symfony config files, etc., will require the cache to be cleared regardless of the environment.
+
+The typical rule of thumb is, if Mautic is not acting as you expect after making changes, try clearing your cache. If you get "class could not be found" or "cannot redeclare class" errors when using the ``cache:clear`` command, manually delete the ``app/cache/ENV`` folder then run the command and/or browse to the site to rebuild.
+
+Cache Bundle
+------------
+
+Enables PSR-6 and PSR-16 caching. Check: `Symfony Cache Component <https://symfony.com/doc/current/components/cache.html>`_
```

Comment:
@fakela please convert this to a link using the 'make link' command within the docs directory, and commit the file that is created. Thanks!